desc/magnetic_fields.py

```diff
@@ -2029,7 +2026,7 @@ def _compute_magnetic_field_from_CurrentPotentialField(
     coords = jnp.atleast_2d(coords)
     if basis == "rpz":
         coords = rpz2xyz(coords)
-    surface_grid = source_grid or LinearGrid(M=30, N=30, NFP=field.NFP)
+    surface_grid = source_grid or LinearGrid(M=60, N=60, NFP=field.NFP)
```

Comment:
would it be better to have this depend on M,N of the surface/potential in some way?

Reply:
we might as well replace instances of `surface_grid` with `source_grid` to be consistent, no need for the variable name change here

Reply:
@f0uriest The problem with doing Rory's suggestion is that a surface current whose potential is purely the secular terms can have a M,N of 0 or 1 or something, but to accurately get the biot savart law you still need a lot of points (at least in my experience), so I figured a large default should be used

Reply:
what about something like `30 + 2*max(self.M, self.M_phi)`? that way you get a reasonable resolution plus it scales as you increase the complexity of the surface